src/GraphQL.NewtonsoftJson/NewtonsoftContractResolver.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Reflection;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace GraphQL.NewtonsoftJson
+{
+    public class NewtonsoftContractResolver : DefaultContractResolver
```

Comment:
The name `NewtonsoftContractResolver` seems strange given that the namespace is already `GraphQL.NewtonsoftJson`. I propose to change it to `ExecutionResultContractResolver` to make the purpose clear.

Reply:
Renamed